src/main/java/main/TaskList.java

```diff
@@ -0,0 +1,179 @@
+package main;
+
+import exception.DukeException;
+import task.*;
+
+import java.util.ArrayList;
+
+/**
+ * This class is used to handle operations regarding the list of tasks
+ * such as adding or deleting
+ */
+public class TaskList {
+
+    /**
+     * Creates TaskList object with the given taskList by the Storage class
+     * @param taskList ArrayList of type Task to be be used
+     */
+    private final ArrayList<Task> taskList;
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Returns size of taskList to be used for loops or other operations
+     * @return size of taskList
+     */
+    public int size() {
+        return taskList.size();
+    }
+
+    /**
+     * Displays the list of tasks in taskList
+     */
+    public String list() {
```

Comment:
Perhaps this could be named getList(), as that is the function of the list() here and it will adhere to Coding Standard